spot_painting/main.py

```diff
@@ -0,0 +1,23 @@
+import turtle as t
+import random
+
+tim = t.Turtle()
+t.colormode(255)
+
+def random_color():
+  r = random.randint(0, 255)
+  g = random.randint(0, 255)
+  b = random.randint(0, 255)
+  random_color = (r, g, b)
```

Comment:
Note you're returning a tuple (essentially a list that you can't change) here, not a list. Not sure whether or not `turtle.color` is sensitive to the data type you pass in, but a tuple is an appropriate data type for this since you won't be changing it for any reason.